tests/helpers.py

```diff
@@ -62,3 +73,85 @@ def dict_to_item(raw,convert_root=True):
         item =  {'N': str(raw)}
 
     return item if convert_root else item['M']
+
+def mock_execution_results_table_entry():
+    random_id = gen_id()
+    execution_id = gen_id()
+    investigation_id = gen_id()
+    date_time = gen_datetimenow()
+    context = {
+            "datetime": date_time,
+            "execution_id": execution_id,
+            "investigation_id": investigation_id,
+            "results": {
+                'artifacts': {
+                    'event': {
+                        'id': random_id,
+                        'created_at': date_time,
+                        'data_types': {},
+                        'details': {
+                            "some": "randon text"
+                        },
+                        'event_type': 'Test integrations',
+                        'event_meta': {},
+                        'investigation_id': investigation_id,
+                        'status_': 'open',
+                        'is_duplicate': False
+                    },
+                    'execution_id': execution_id
+                },
+                "errors": {},
+                "results": {}
+            }
+        }
+    results_table_name = os.environ['SOCLESS_RESULTS_TABLE']
+    client = boto3.client('dynamodb')
+    client.put_item(
+        TableName=results_table_name,
+        Item=dict_to_item(
+            context,
+            convert_root=False
+        )
+    )
+    return {'id': random_id,
+            "execution_id": execution_id,
+            "investigation_id": investigation_id,
+            "datetime": date_time,
+            'context': context}
+
+def mock_execution_results_table_entry_with_sfn():
```

Comment:
This looks more like a mock of the step functions context object, not an execuition_results_table mock. Is that what its supposed to be? If so please rename the function as such